validation/src/test/scala/hmda/validation/rules/lar/validity/V210Spec.scala

```diff
@@ -0,0 +1,64 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V210Spec extends LarEditCheckSpec {
+  property("All applications must pass") {
```

Comment:
Seeing "application" in the spec descriptions instead of "LAR" confused me for a few minutes. Tell me about your decision to use that wording?